components/Header.tsx

```diff
@@ -144,6 +146,49 @@ export const Header = ({
                 {linkName}
               </Link>
             ))}
+            {drawerVisible ? (
+              <>
+                <Link
+                  href="https://www.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
+                >
+                  Forums
+                </Link>
+
+                <Link
+                  href="https://portal.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
+                >
+                  Portal
+                </Link>
+
+                <Link
+                  href="https://cards.simulationhockey.com"
+                  _hover={{ textDecoration: 'none' }}
+                  className={LINK_CLASSES}
+                >
+                  Cards
+                </Link>
+              </>
+            ) : (
+              <Menu>
+                <MenuButton className={LINK_CLASSES}>More</MenuButton>
+                <MenuList>
+                  <MenuDivider />
```

Comment:
Why do we have a menu divider here? This looks weird and adds too much top padding
<img width="119" alt="Screenshot 2024-06-17 at 4 49 40 PM" src="https://github.com/esilverm/shl-index-ui/assets/24704789/3a859f52-c37a-41bf-9efa-c97a519c3347">

```suggestion
```

Reply:
it was on the move from the portal -> index. Deleted it on the first pass. Forgot about it while brining it back in. Deleted